### ISSUE TITLE
Nav menu tabbing and menu drawer focus order

### ISSUE THREAD

**Issue by aviannamiller** [omeka-s-themes/freedom#40]
For keyboard users, there is a tabbing issue with menu navigation. Tabbing to a parent activates it automatically and opens its list of child links, which must all be tabbed through in order to navigate to the next parent. A [similar issue](https://github.com/omeka-s-themes/default/issues/42) presented itself for default:

> The issue ... is well-summarized by [W3C's comments on Keyboard Users](https://www.w3.org/WAI/tutorials/menus/flyout/#keyboard-users): 
> 
> "Submenus should not open when using the tab key to navigate through the menu, as keyboard users would then have to step through all submenu items to get to the next top-level item."
> 
> VoiceOver is behaving in such a way that a user must tab through all child menu items, which are read out, in order to get to the next parent (not sure if the same is true when tested with NVDA or JAWS). To remedy this, it seems that using a button as a toggle (see [Use button as toggle](https://www.w3.org/WAI/tutorials/menus/flyout/#use-button-as-toggle)) would allow for screenreader users to navigate from parent to parent without the readout of each child item unless the button is activated.
> 

 _Originally posted by @aviannamiller in [#42](https://github.com/omeka-s-themes/default/issues/42#issuecomment-2695730019)_

There also seem to be some general focus order issues with the menu drawer.

When the menu drawer is open, focus moves immediately to content behind the open menu that should otherwise be hidden while the menu is open, followed by the "Close Menu" link, then the parent links:
<img width="577" alt="Screenshot of menu drawer with numbers and lines illustrating focus order overlaying list items" src="https://github.com/user-attachments/assets/fa2d6d60-4239-469b-9001-3be07511200e" />
Besides elements gaining focus that technically shouldn't while the menu is activated, this focus order seems illogical but also further emphasizes issues with the non-focusable toggle button pointed out in #37. Technically, that button should be the trigger element that focus returns to once list items are tabbed through, which may deem the "Close Menu" link redundant.

When the menu drawer is closed, its list items still gain focus while tabbing through the main page content. These elements should be hidden when tabbing through the page with the menu collapsed, especially as tabbing through the menu already occurs through the nav header bar, so this is redundant.

Another issue: using keyboard to navigate through and open nested child links in the menu drawer eventually leads to empty containers with focus moved to elements that are no longer on the page, as in the following example:
<img width="758" alt="Screenshot of menu drawer, which is presenting as an empty container. Focus is on an unidentified element to the left of the screen" src="https://github.com/user-attachments/assets/3259ebe1-b624-4ec8-a541-3bd6d9bba067" />
It appears that these focusable elements are list item links that are on previous menu pages and technically should not be able to gain focus once one has navigated to following menu pages.

**Comment by nelsonamaya82** [comment]
@aviannamiller this is fixed in the `master` branch, please check again and let me know if it works as expected. Thanks.

**Comment by nelsonamaya82** [comment]
@aviannamiller this is fixed in the `master` branch. I applied all the adjustments we applied and tested on Lively Classic, so it now works exactly the same as in Classic. Please retest and let me know if it works.

**Comment by aviannamiller** [comment]
Thank you, this works well!
